PRS1001-2000/PRS1277Abraham.xml

```diff
@@ -32,10 +32,13 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
          </xi:include>
       </encodingDesc>
       <profileDesc>
-         <langUsage><language ident="en">English</language></langUsage>
+         <langUsage>
+           <language ident="en">English</language>
+           <language ident="gez">Gǝʿǝz</language>
+         </langUsage>
          <textClass>
             <keywords>
-               <term key="saint"/>
+               <term key="martyr"/>
```

Comment:
I would maybe keep saint, as they are not mutually exclusive. What do you think?